<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Cutcell</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="translationLabel">
           <property name="text">
            <string>&amp;Translation</string>
           </property>
           <property name="buddy">
            <cstring>translationXSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0,1,0,1">
           <item>
            <widget class="QLabel" name="transXLabel">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="translationXSpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="transYLabel">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="translationYSpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="transZLabel">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="translationZSpinBox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="scalingLabel">
           <property name="text">
            <string>&amp;Scaling</string>
           </property>
           <property name="buddy">
            <cstring>scalingSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="scalingSpinBox">
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="offFileLabel">
           <property name="text">
            <string>O&amp;FF file</string>
           </property>
           <property name="buddy">
            <cstring>offFileLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="offFileLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="offOpenButton">
             <property name="text">
              <string>Open...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1,0,0,1,0,1">
         <item>
          <widget class="QLabel" name="cellSizeXLabel">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="cellSizeXDoubleSpinBox">
           <property name="maximum">
            <double>9999999.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cellSizeCubicButton">
           <property name="text">
            <string>Cubic =&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cellSizeYLabel">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="cellSizeYDoubleSpinBox">
           <property name="maximum">
            <double>9999999.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cellSizeZLabel">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="cellSizeZDoubleSpinBox">
           <property name="maximum">
            <double>9999999.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="gridSizeLabel">
         <property name="text">
          <string>G&amp;rid size</string>
         </property>
         <property name="buddy">
          <cstring>gridSizeXSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1,0,0,1,0,1">
         <item>
          <widget class="QLabel" name="gridSizeXLabel">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="gridSizeXSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gridSizeCubicButton">
           <property name="text">
            <string>Cubic =&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gridSizeYLabel">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="gridSizeYSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gridSizeZLabel">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="gridSizeZSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="originLabel">
         <property name="text">
          <string>&amp;Origin</string>
         </property>
         <property name="buddy">
          <cstring>originXDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1,0,0,1,0,1">
         <item>
          <widget class="QLabel" name="originXLabel">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="originXDoubleSpinBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="originCubicButton">
           <property name="text">
            <string>Cubic =&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="originYLabel">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="originYDoubleSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="originZLabel">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="originZDoubleSpinBox"/>
         </item>
        </layout>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="cgnsFileLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="cgnsFileLabel">
         <property name="text">
          <string>CG&amp;NS file</string>
         </property>
         <property name="buddy">
          <cstring>cgnsFileLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="cellSizeLabel">
         <property name="text">
          <string>&amp;Cell size</string>
         </property>
         <property name="buddy">
          <cstring>cellSizeXDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="cgnsOpenButton">
         <property name="text">
          <string>Open...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>&amp;Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
